app/components/nav/hamburgerMenu.js

```diff
@@ -9,7 +9,7 @@ const HamburgerMenu = ({ isOpen, onClick }) => (
     <div className="last" />
     <style jsx>
       {`
-        @media (max-width: 768px) {
+        @media (max-width: 874px) {
```

Comment:
i still see a cutoff:
<img width="801" alt="screen shot 2019-01-29 at 22 14 58" src="https://user-images.githubusercontent.com/5598081/51937466-5fb22600-2413-11e9-9c99-af38d695fb86.png">

i think `@media (max-width: 885px) ` would work best.

Also, what do you think about adding this value in `constants/index.js`? Could be `mediaQueries.HAMBURGER_MENU = '885px'`...